### DISCUSSION TITLE
Does Graphql support actions/workflow/runs/jobs collect?

### DISCUSSION THREAD

**Discussion by likyh** [community/community#37974] [Apps, API and Webhooks] [answered]
I want to collect runs/jobs from graphql.
The same APIs in Restful are `repos/{Owner}/{Repo}/actions/runs` and `repos/{Owner}/{Repo}/actions/runs/{RunId}/jobs`.

I can only find workflow in `commit -> checkSuites -> workflowRun -> workflows` and it not works.

**Comment by LangLangBart** [comment, marked as answer]
The last request is 2 years old, no action API for GraphQL. --> https://github.com/community/community/discussions/24493
